platform/ios/src/MGLMapView.h

```diff
@@ -416,7 +416,7 @@ MGL_EXPORT IB_DESIGNABLE
  transition. If you don’t want to animate the change, use the
  `-setUserLocationVerticalAlignment:animated:` method instead.
  */
-@property (nonatomic, assign) MGLAnnotationVerticalAlignment userLocationVerticalAlignment;
+@property (nonatomic, assign) MGLAnnotationVerticalAlignment userLocationVerticalAlignment __attribute__((deprecated("Use -userLocationAnchorForMapView delegate method instead.")));
 
 /**
  Sets the vertical alignment of the user location annotation within the
```

Comment:
Should there be an instance method for updating the offset spontaneously? Or is it sufficient to rely on the delegate method to be called once per location update?